modules/README.md

```diff
@@ -153,6 +162,10 @@ Modules that are still in development and less stable are labeled with the
   Creates a Slurm login node using [slurm-gcp-version-5].
 * **[schedmd-slurm-gcp-v5-hybrid]** ![community-badge] ![experimental-badge] :
   Creates hybrid Slurm partition configuration files using [slurm-gcp-version-5].
+* **[schedmd-slurm-gcp-v6-controller]** ![community-badge] ![experimental-badge]:
```

Comment:
Links do not point to anything/are broken. Please fix before merging.

Reply:
Thank you for spotting . Fixed.